src/onnxruntime_utils.h

```diff
@@ -37,13 +37,50 @@ namespace triton { namespace backend { namespace onnxruntime {
 
 extern const OrtApi* ort_api;
 
-#define RESPOND_ALL_AND_RETURN_IF_ERROR(RESPONSES, RESPONSES_COUNT, X) \
-  do {                                                                 \
-    TRITONSERVER_Error* raarie_err__ = (X);                            \
-    if (raarie_err__ != nullptr) {                                     \
-      SendErrorForResponses(RESPONSES, RESPONSES_COUNT, raarie_err__); \
-      return;                                                          \
-    }                                                                  \
+#define RESPOND_ALL_AND_RETURN_IF_ERROR(                            \
+    REQUESTS, REQUEST_COUNT, RESPONSES, S)                          \
+  do {                                                              \
+    TRITONSERVER_Error* raarie_err__ = (S);                         \
+    if (raarie_err__ != nullptr) {                                  \
+      SendErrorForResponses(RESPONSES, REQUEST_COUNT, raarie_err__);\
+      for (uint32_t r = 0; r < REQUEST_COUNT; ++r) {                \
```

Comment:
nit: you should create a utility function similar to SendErrorForResponses since this for loop is present in 2 places in this file